server/ntb/macros/nob_NO_translate_macro.py

```diff
@@ -80,6 +80,10 @@ def nob_NO_translate_macro(item, **kwargs):
 def get_user_preference_params():
     user = get_user()
     user_macro_preferences = user.get("user_preferences", {}).get("macro_config", {})
+
+    if not user_macro_preferences:
+        return []
+
     field_param = user_macro_preferences.get("fields").get("Formval nynorskrobot", "")
```

Comment:
this would still fail if `user_macro_preferences` would be like `{"fields": None}`, not sure if that's possible

Reply:
maybe, we can handle this as well. Thanks for pointing it out! 👍